### PR TITLE
Maxmind will have ftRegion as well and geo transform will use alpha2

### PR DESCRIPTION
**Pull request by akilananb**
Maxmind will have ftRegion as weel and geo transform will use alpha2 code instead alpha3

### CODE REVIEW

File: server/utils/ft-regions.js [hunk 1]
```diff
@@ -1,243 +1,243 @@
...
-	['IRN', 'MIDDLEEAST'],
-	['IRQ', 'MIDDLEEAST'],
-	['EGY', 'MIDDLEEAST'],
-	['BHR', 'MIDDLEEAST'],
-	['DZA', 'MIDDLEEAST'],
-	['PSE', 'MIDDLEEAST'],
-	['IND', 'INDIA'],
-	['LKA', 'INDIA'],
-	['NPL', 'INDIA']
+	["GB", "UK"],
```
Comment (akilananb) [inline server/utils/ft-regions.js]: I created a manual Script To map ISO3 to ISO2. But These Codes were missing ZAR,YUG,REU,FXX,TMP,ANT added manually

File: tests/server/transforms/maxmind-api.spec.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ describe('MaxMind', function() {
 		
 		this.mitm.on("request", (req, res) => {
 			res.statusCode = 200;
-			res.end(JSON.stringify({"query": { }, "result": { "countryName": "Germany" }}));
+			res.end(JSON.stringify({"query": { }, "result": { "countryName": "Germany","country":{"code":"DE"}}}));
```
Comment (commuterjoy) [inline tests/server/transforms/maxmind-api.spec.js]: This is slightly wrong. The response from MaxMind is ...  

```
{
    "query": "52.28.221.96",
    "result": {
        "countryCode": "DE",
        "countryName": "Germany",
        "regionName": "Hessen",
        "city": "Frankfurt",
        "postalCode": "60438",
        "latitude": 50.11670000000001,
        "longitude": 8.683300000000003,
        "continentCode": "EU",
        "organisation": "E.I. du Pont de Nemours and Co.",
        "id": "9ff1967c3ac3e3c9244c42a61ea53629",
        "ip": "52.28.221.96"
    }
}
```

File: tests/server/transforms/maxmind-api.spec.js [hunk 3]
```diff
@@ -90,4 +104,18 @@ describe('MaxMind', function() {
...
+	it('Do not add FT Region if country code is not available', done => {
+		this.mitm.on("request", (req, res) => {
+			res.statusCode = 200;
+			res.end(JSON.stringify({"query": { }, "result": { "countryName": "Germany"}}));
+		})
+
+		sinon.stub(e, 'pluck').withArgs('device.ip').returns('132.123.132.123');
+		maxmind(e)
+			.then(function (res) {
+				expect(res.ftRegion).to.be.a('Undefined');
```
Comment (commuterjoy) [inline tests/server/transforms/maxmind-api.spec.js]: `expect(res.ftRegion).to.not.exist;` is probably simpler

File: server/transforms/maxmind-api.js [hunk 4]
```diff
@@ -50,7 +49,11 @@ module.exports = function (event) {
 			if (status !== 200) {
 				return {};
 			}
-
+			if(res.result.country && res.result.country.code)
+			{
+				res.result['ftRegion']=regions.get(res.result.country.code);
+				console.log('Added '+ res.result['ftRegion'] +" as FT Region for "+res.result.country.code);
```
Comment (commuterjoy) [inline server/transforms/maxmind-api.js]: The convention I've been using is to prefix all the logging with the name of the module...

`console.log('transforms/maxmind', ip, status, JSON.stringify(res));`

**Comment by commuterjoy** [comment]
+1 good work, lets merge and deploy this on Monday.

**Pull request merged**